client/src/components/App.js

```diff
@@ -1,70 +1,72 @@
-import React, { Component } from 'react';
-import NavBar from './NavBar';
-import Temperature from './Temperature'
-import Login from './Login';
-import 'bootstrap/dist/css/bootstrap.min.css';
-import '../css/App.css';
-import { appConfig } from '../utils/constants'
-import { UserSession } from 'blockstack';
-import Button from 'react-bootstrap/Button';
-import { configure, User, getConfig } from 'radiks';
-import Patient from '../models/patient'
+import React, { Component } from "react";
+import NavBar from "./NavBar";
+import Temperature from "./Temperature";
+import Login from "./Login";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "../css/App.css";
+import { appConfig } from "../utils/constants";
+import { UserSession } from "blockstack";
+import Button from "react-bootstrap/Button";
+import { configure, User, getConfig } from "radiks";
+import Patient from "../models/patient";
 
 const userSession = new UserSession({
   appConfig: appConfig
-})
+});
 
 export default class App extends Component {
-
   constructor(props) {
-    super(props)
-    this.userSession = new UserSession({ appConfig })
+    super(props);
+    this.userSession = new UserSession({ appConfig });
   }
 
   async componentDidMount() {
+    if (!process.env.REACT_APP_QA_URL) {
+      return;
+    }
```

Comment:
I wonder if, since we're just building this out for the time being to connect to the QA database just for devs, if we should throw some kind of `alert` before returning reminding devs to be sure to set their environment variables?

Reply:
Let's throw the alert() to remind devs. SInce they aren't familiar too much with codebase, it would be a helpful reminder.

Reply:
added